ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/ExecutionEnvironmentSection.java

```diff
@@ -113,7 +116,8 @@ public void dispose() {
 	}
 
 	public ExecutionEnvironmentSection(PDEFormPage page, Composite parent) {
-		super(page, parent, Section.DESCRIPTION, new String[] { //
+		super(page, parent, Section.DESCRIPTION, new String[] {
```

Comment:
Nitpick: Please preserve the comment to prevent changes by future formatting.
```suggestion
		super(page, parent, Section.DESCRIPTION, new String[] { //
```